### ISSUE TITLE
PaperScan

### ISSUE THREAD

**Issue by alexmyczko** [trunkmaster/nextspace#29]
There's a scanning software gnustep application you might be interested in including?
https://github.com/poroussel/paperscan

**Comment by trunkmaster** [comment]
It maybe interesting but definitely not with the highest priority. Do you have screenshots of it?

**Comment by alexmyczko** [comment]
Not yet, https://github.com/poroussel/paperscan/issues/2

Meanwhile I'm struggling linking libopenmpt-modplug(-dev) with GNUMakefile:
https://github.com/alexmyczko/ModPlugPlay.app
and having issues with Gorms' Resource management of tiff or png images...

But the VolumeManager works quite well, maybe you want that? https://github.com/alexmyczko/VolumeControl.app

**Comment by trunkmaster** [comment]
Yes! :+1:  I need volume control for the Sound Preferences in Preferences.app. Thank you.
I see VolumeControl uses OSS and ALSA. Do you think about integration with PulseAudio like `pavucontrol` tool?

**Comment by alexmyczko** [comment]
I accept patches :) 
OSS is for FreeBSD and ALSA for Linux.

**Comment by trunkmaster** [comment]
Okay, I understand you. :) After Workspace features implementation I plan to return to Preferences. And then... ;)

**Comment by trunkmaster** [comment]
@alexmyczko, I've created enhanced version of ALSA mixer: shows ALSA elements registered by sound card driver, balance control, 3 classes - ALSA (mixer window and popup buttons), ALSAElement(mixer control element and it's properties), ALSACard (sound card properties, mixer and array of ALSAElement). Check it out from here: https://github.com/trunkmaster/nextspace/tree/master/Frameworks/Tests/Mixer.

The next goal is PulseAudio mixer.
